### ISSUE TITLE
Add social buttons to promote wafflejs

### ISSUE THREAD

**Issue by gcochard** [wafflejs/wafflejs.github.io#143]
We should add some social share buttons so attendees can say "_I'm_ going, what about **you**???".

**Comment by visnup** [comment]
yeah, I've been wanting to do this for a while too. @billyroh any idea where they'd go?
